.github/workflows/tests.yml

```diff
@@ -40,11 +40,13 @@ jobs:
                 image: rabbitmq:3.8.3
                 ports:
                     - 5672:5672
-
         steps:
             -   name: Checkout
                 uses: actions/checkout@v2
 
+            - name: Populate LDAP
```

Comment:
I believe the LDAP server should be a "service" instead of a "step"

Reply:
The action should populate the server. But, I couldn't let them communicate.... Then I was wondering if I could only have an action with a persistent openldap server... I'm not sure if it is possible.... Just experimenting.